integration-tests/src/test/java/org/apache/fineract/integrationtests/SavingsAccountTransactionTest.java

```diff
@@ -211,6 +213,64 @@ public void testConcurrentSavingsBatchTransactions() {
 
         }
         this.datatableHelper.deleteDatatable(datatableName);
+        try {
+            for (Future<?> result : results) {
+                assertNull(result.get());
+            }
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+        log.info("\nFinished all threads");
+    }
+
+    // @Test
```

Comment:
ACK: this test is commented out temporarily till it got fixed (locally and on GA behaves differently)